corehq/apps/geospatial/static/geospatial/js/gps_capture.js

```diff
@@ -130,6 +130,9 @@ hqDefine("geospatial/js/gps_capture",[
             return !self.showLoadingSpinner() && !self.hasError();
         });
 
+        self.createCaseEnabled = ko.observable(false);
+        self.createCaseError = ko.observable(false);
```

Comment:
nit: I assume this is the error message and so the default value should be empty text or undefined/null and not a boolean.

Reply:
oh it is a boolean flagging if there is an error, may be a different name?
good to ensure its reset where needed.

Reply:
I can see how this might be confusing. I think something like `hasCreateCaseError` will make it more explicit that this is a boolean.